adapters/p2p2core/transaction_test.go

```diff
@@ -0,0 +1,63 @@
+package p2p2core_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/NethermindEth/juno/adapters/core2p2p"
+	"github.com/NethermindEth/juno/adapters/p2p2core"
+	transactiontestutils "github.com/NethermindEth/juno/adapters/testutils"
+	"github.com/NethermindEth/juno/core"
+	"github.com/NethermindEth/juno/core/felt"
+	"github.com/NethermindEth/juno/sync"
+	"github.com/NethermindEth/juno/utils"
+	"github.com/starknet-io/starknet-p2pspecs/p2p/proto/common"
+	synctransaction "github.com/starknet-io/starknet-p2pspecs/p2p/proto/sync/transaction"
+	"github.com/stretchr/testify/require"
+)
+
+var TransactionBuilder = transactiontestutils.TransactionBuilder[
+	sync.Transaction, *synctransaction.TransactionInBlock]{
```

Comment:
This is to avoid having 2 different levels having the same level of indentation.
```suggestion
var TransactionBuilder = transactiontestutils.TransactionBuilder[
	sync.Transaction,
	*synctransaction.TransactionInBlock,
]{
```